conans/test/functional/old/test_migrations.py

```diff
@@ -60,10 +59,10 @@ def test_migrate_revision_metadata(self):
         client.run("search")  # This will fire a migration
 
         metadata_ref1 = client.cache.package_layout(ref).load_metadata()
-        self.assertEqual(metadata_ref1.recipe.revision, "2e48797069e65568befd81854aa8aaf0")
+        self.assertEqual(metadata_ref1.recipe.revision, "f9e0ab84b47b946f4c7c848d8f82d14e")
```

Comment:
Why does the revision change here? Different conanfile content?

Reply:
Yes, the generated `conanfile.py` is different (may have the same content, but different order or more blank lines), so the hash is different and so the revision.